src/app/category-chart/overview/category-chart-overview-sample.component.html

```diff
@@ -0,0 +1,19 @@
+<div>
+	<span>Chart Type:</span>
+	<select [(ngModel)]="chartType">
+		<option>Auto</option>
+		<option>Line</option>
+		<option>Column</option>
+		<option>Point</option>
+		<option>Spline</option>
+		<option>SplineArea</option>
+		<option>StepArea</option>
+		<option>StepLine</option>
+		<option>Waterfall</option>
+	</select>
+</div>
+<igx-category-chart height="500px" width="100%"
```

Comment:
width 100% is going to look pretty wide in a lot of circumstances possibly, with a static height. What is the recommendation here for if we are hosted in a embedded stackblitz panel? I assume we may want both width and height to be 100%, but in this case you'd need to make sure:

```css
html, body {
   height: 100%;
}
```

Reply:
I see we have `.sample-wrapper { height: 100%; }`, but it looks like it's used in each individual sample.  Which is a bit strange, but I can make that change.  Is this repo supposed to be more of a collection of samples as opposed to being a samples browser?

I would also think we would want the html and body height set on a higher application-level, but I don't currently see any CSS doing that.